community/server/src/test/java/org/neo4j/server/web/ExecutionTimeLimitTest.java

```diff
@@ -1,108 +0,0 @@
-/*
- * Copyright (c) 2002-2015 "Neo Technology,"
- * Network Engine for Objects in Lund AB [http://neotechnology.com]
- *
- * This file is part of Neo4j.
- *
- * Neo4j is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-package org.neo4j.server.web;
-
-import com.sun.jersey.api.client.Client;
-import com.sun.jersey.api.client.UniformInterfaceException;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Ignore;
-import org.junit.Test;
-
-import org.neo4j.graphdb.Node;
-import org.neo4j.kernel.GraphDatabaseAPI;
-import org.neo4j.server.WrappingNeoServerBootstrapper;
-import org.neo4j.server.configuration.Configurator;
-import org.neo4j.server.configuration.ServerConfigurator;
-import org.neo4j.test.ImpermanentGraphDatabase;
-
-@Ignore("This feature has to be done some other way")
-public class ExecutionTimeLimitTest
```

Comment:
+1, I'll add it to the remoting wall
On Jun 17, 2015 7:14 AM, "Davide Grohmann" <notifications@github.com> wrote:

> In
> community/server/src/test/java/org/neo4j/server/web/ExecutionTimeLimitTest.java
> <https://github.com/neo4j/neo4j/pull/4815#discussion_r32619788>:
>
> > -import com.sun.jersey.api.client.Client;
> > -import com.sun.jersey.api.client.UniformInterfaceException;
> > -import org.junit.After;
> > -import org.junit.Before;
> > -import org.junit.Ignore;
> > -import org.junit.Test;
> > -
> > -import org.neo4j.graphdb.Node;
> > -import org.neo4j.kernel.GraphDatabaseAPI;
> > -import org.neo4j.server.WrappingNeoServerBootstrapper;
> > -import org.neo4j.server.configuration.Configurator;
> > -import org.neo4j.server.configuration.ServerConfigurator;
> > -import org.neo4j.test.ImpermanentGraphDatabase;
> > -
> > -@Ignore("This feature has to be done some other way")
> > -public class ExecutionTimeLimitTest
>
> You probably know better than me what to write in those cards. Mind to
> create them?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/neo4j/neo4j/pull/4815/files#r32619788>.
>